openfisca_us/tests/policy/baseline/gov/states/md/tax/income/credits/md_ctc.yaml

```diff
@@ -0,0 +1,22 @@
+- name: MD Ctc
+  period: 2020
+  input:
+    people:
+      head:
+        age: 36
+        is_tax_unit_head: true
+        is_tax_unit_dependent: false
+        employment_income: 2_000
```

Comment:
Use variables in the formula for unit tests, so skip info on the head and put `adjusted_gross_income` in the tax unit.